Automattic/wp-calypso
# Add support for Pocket Cast embed to Reader streams

This PR will add support for Pocket Cast embeds to the Reader streams.

This treats the embed as a video iframe type. The only issue I found was that the Pocket Cast embed iframes have no width or height dimensions defined so we need to handle this when rendering a `ReaderFeaturedVideo`.

This PR also adds a default Pocket Cast SVG image to use as a thumbnail until a thumbnail is available from the oembed request.

**Before:**

<img width="622" alt="Screenshot 2023-12-15 at 19 12 17" src="https://github.com/Automattic/wp-calypso/assets/5560595/e008d572-d6b5-4400-bddc-7a21f72864e8">

**After:**

<img width="629" alt="Screenshot 2023-12-15 at 19 09 52" src="https://github.com/Automattic/wp-calypso/assets/5560595/a691ef9d-1d75-4c83-ba5f-43024e1a13ba">


### Testing
* Go to calypso live
* Go to the following feed - read/feeds/7025646
* Click on the Pocket cast image
* You should be able to hear the Pocket cast embed
